ui/ducks/confirm-alerts/confirm-alerts.ts

```diff
@@ -0,0 +1,116 @@
+import { Severity } from '../../helpers/constants/design-system';
+
+export type Alert = {
```

Comment:
As this is the central definition of an `Alert`, should we add some JSDoc on the type itself and the properties?